include/picongpu/algorithms/FieldToParticleInterpolation.hpp

```diff
@@ -48,8 +48,8 @@ struct FieldToParticleInterpolation
 
     static constexpr int lowerMargin = supp / 2 ;
     static constexpr int upperMargin = (supp + 1) / 2;
-    typedef typename pmacc::math::CT::make_Int<simDim,lowerMargin>::type LowerMargin;
-    typedef typename pmacc::math::CT::make_Int<simDim,upperMargin>::type UpperMargin;
+    using LowerMargin = typename pmacc::math::CT::make_Int<simDim,lowerMargin>::type;
+    using UpperMargin = typename pmacc::math::CT::make_Int<simDim,upperMargin>::type;
```

Comment:
I think these are not dependent, also in other cases with `make_Int<simDim, ...>`  (note: in some places it is `make_Int<T_dim, ...>`, which is ofc dependent).